moto/apigateway/responses.py

```diff
@@ -387,13 +387,15 @@ def integration_responses(self, request, full_url, headers):
             elif self.method == "PUT":
                 selection_pattern = self._get_param("selectionPattern")
                 response_templates = self._get_param("responseTemplates")
+                content_handling = self._get_param("contentHandling")
```

Comment:
According to [the docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/apigateway.html#APIGateway.Client.put_integration_response), this param is not mandatory